### PR TITLE
Feature/new failure causes

### PR DESCRIPTION
**Pull request by wgromniak**

### CODE REVIEW

**Comment by wgromniak** [comment]
@mchalapuk, I'll release to maven central once you approve.

**Comment by zarazek** [comment]
A little confusion about version number. Why is it 0.7.0 and not 0.6.0?

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ using Java.
 
 ## Important!
 
+* Next to this documentation, please read the [general documentation](https://quedex.net/doc/api) of our WebSocket API.
```
Comment (mchalapuk) [inline README.md]: I don't think this should be the first point.

Comment (wgromniak) [reply README.md]: Hmm, why? I've thought that this is something that should be read before this documentation.

**Comment by wgromniak** [comment]
@zarazek, it'll be published to maven central as 0.6.0.

**Pull request merged**